packages/agreement-process/src/services/agreementService.ts

```diff
@@ -397,6 +396,12 @@ export function agreementServiceBuilder(
         consumer
       );
 
+      /* TODO: 
+        FIX the newstate by flag should considerer the suspendedByPlatformFlag, 
+        it's necesary to evaluate again the falg, if it is true, stop submit flow and return 
+        event type AgreementSetToMissingCertifiedAttributesByPlatform, after save event 
+        in the eventstore to update agreemnt's state to MISSING_CERTIFIED_ATTRBUTES
+      */
```

Comment:
🧹  you can remove this comment, I am already doing it in PR #578 and also tracked it on JIRA

Reply:
OK I trust you, comment removed